nav2_waypoint_follower/CMakeLists.txt

```diff
@@ -18,9 +23,11 @@ find_package(rclcpp_lifecycle REQUIRED)
 find_package(nav_msgs REQUIRED)
 find_package(nav2_msgs REQUIRED)
 find_package(nav2_util REQUIRED)
+find_package(nav_2d_utils REQUIRED)
 find_package(tf2_ros REQUIRED)
 find_package(nav2_core REQUIRED)
 find_package(pluginlib REQUIRED)
+find_package(robot_localization REQUIRED)
```

Comment:
You don't need to find package or include it in dependencies to link against. We just need RL for the services it exposes, no need to link against it. As long as its in the package.xml, you're good to go.

Reply:
This needs to be linked we need a header for the service message
`#include "robot_localization/srv/from_ll.hpp"`

Reply:
Ah yes, thanks